# reaching motion params

reach_rate : 5.0

approach_height : 3.0
descending_height : 2.0
approach_margin : 0.05
height_margin : 0.05
covering_pre_height : 1.0
covering_post_height : 0.2
covering_move_dist : 1.0

